src/back/routes/events.ts

```diff
@@ -14,8 +17,17 @@ export default routes((route) => {
   route.post('/events/discourse/new', handleAPI(discourseComment)) //TODO: deprecate
 })
 
-async function getLatestEvents() {
-  return await EventsService.getLatest()
+async function getLatestEvents(req: Request) {
+  const { event_type } = req.query
+  const filters: Record<string, unknown> = {}
+  if (event_type) {
+    filters.event_type = isArray(event_type) ? event_type : [event_type]
+  }
+  const parsedEventTypes = EventFilterSchema.safeParse(filters)
+  if (!parsedEventTypes.success) {
+    throw new Error('Invalid event types: ' + parsedEventTypes.error.message)
+  }
```

Comment:
Maybe this could be in a `validateEventTypesFilters` and that function could have a test. Just to be sure.